plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -35,6 +35,30 @@ void testShowTables()
         assertUpdate("CREATE TABLE faker.default.test (id INTEGER, name VARCHAR)");
         assertTableColumnNames("faker.default.test", "id", "name");
     }
+    @Test
+    void testRenameTable()
+    {
+       assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
+       assertUpdate("CREATE TABLE faker.default.original_table (id INTEGER, name VARCHAR)");
+       assertQuery("SHOW TABLES FROM faker.default", "VALUES 'original_table'");
+       assertUpdate("ALTER TABLE faker.default.original_table RENAME TO renamed_table");
+       assertQuery("SHOW TABLES FROM faker.default LIKE 'renamed_table'", "VALUES 'renamed_table'");
+       assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
+       assertUpdate("DROP TABLE faker.default.renamed_table");
+    }
+    @Test
+    void testAlterTableSetProperties() 
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'test_table'", "SELECT '' WHERE false");
+        assertUpdate("CREATE TABLE faker.default.test_table (id INTEGER, name VARCHAR)");
+        assertQuery("SHOW TABLES FROM faker.default", "VALUES 'test_table'");
+        assertUpdate("ALTER TABLE faker.default.test_table SET PROPERTIES comment = 'This is a test table'");
```

Comment:
The connector doesn't support setting `comment` table property. It should be `null_probability` or `default_limit`.